### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by outscale-hmi**
This PR is to update:
- Helm from 3.10.3 to 3.16.1
- k8s version from 1.24.8 to 1.30.5 and from "v1.26.11-rancher2-1" to "v1.30.5-rancher1-1"
- update image_id
- rke version from 1.4.4 to 1.6.3

### CODE REVIEW

**Comment by outscale-rce** [comment]
rke version from 1.4.4 to 1.6.2

File: terraform.tfvars [hunk 1]
```diff
@@ -3,23 +3,23 @@
...
 
 # Ubuntu Image, check latest on https://docs.outscale.com/en/userguide/Official-OMIs-Reference.html
-image_id                  = "ami-9b41b457" # Ubuntu-22.04-2023.02.21-0 (created: 2023-02-21) on us-east-2
-control_plane_vm_type     = "tinav6.c4r8p1"
+image_id                  = "ami-a3ca408c" # Ubuntu-22.04-2023.02.21-0 (created: 2023-02-21) on us-east-2
+control_plane_vm_type     = "tinav4.c4r16p2"
 control_plane_count       = 1
 control_plane_volume_type = "io1"
-control_plane_volume_size = 15
+control_plane_volume_size = 120
```
Comment (outscale-rce) [inline terraform.tfvars:10]: Why using such a big volume ?

File: github_actions/deploy_cluster/action.yml [hunk 2]
```diff
@@ -21,11 +21,11 @@ inputs:
   rke_version:
     description: 'Version of rke to use'
     required: false
-    default: "v1.5.2"
+    default: "v1.6.2"
```
Comment (outscale-rce) [inline github_actions/deploy_cluster/action.yml:24]: Why are you using version 1.6.2 instead of 1.6.3, which is the latest version?

Comment (outscale-hmi) [reply github_actions/deploy_cluster/action.yml:24]: Here the k8s supported bu rke 1.6.2: 
- v1.27.16-rancher1-1
- v1.28.14-rancher1-1
- v1.29.9-rancher1-1
- v1.30.5-rancher1-1 (default) 
I can also update update k8s to a more recent version if we want

Comment (outscale-rce) [reply github_actions/deploy_cluster/action.yml:24]: Ok, so stick to this one !

File: github_actions/deploy_cluster/action.yml [hunk 3]
```diff
@@ -89,7 +89,7 @@ inputs:
   image_id:
     description: 'Image ID used for all VMs'
     required: false
-    default: "ami-9b41b457"
+    default: "ami-1111f1b5"
```
Comment (outscale-rce) [inline github_actions/deploy_cluster/action.yml:92]: Are you sure about this ID ?

File: terraform.tfvars [hunk 4]
```diff
@@ -19,7 +19,7 @@ bastion_volume_type       = "io1"
 bastion_volume_size       = 15
 bastion_iops              = 1500
 cluster_name              = "phandalin"
-rke_version               = "v1.4.4"
-kubernetes_version        = "v1.25.6-rancher4-1" # See available version https://github.com/rancher/rke/releases
-will_install_ccm          = false                # Set to true if the CCM of osc will be installed
+rke_version               = "v1.6.2"
+kubernetes_version        = "v1.30.3-rancher1-1"
```
Comment (outscale-rce) [inline terraform.tfvars:23]: Isn't it v1.30.4-rancher1-1?

File: github_actions/deploy_cluster/action.yml (hunk 3 above)
Comment (outscale-hmi) [reply github_actions/deploy_cluster/action.yml:92]: Yes 
https://docs.outscale.com/fr/userguide/R%C3%A9f%C3%A9rence-des-OMI-officielles.html#_us-east-2
the [Ubuntu-22.04-2023.12.04-0](https://docs.outscale.com/fr/userguide/Ubuntu-22.04-2023.12.04-0.html) in us-east-2

File: terraform.tfvars (hunk 4 above)
Comment (outscale-hmi) [reply terraform.tfvars]: I update both of them (rke%_version and k8s_version to the latest to support 1.30.5 and rke 1.6.3

**Review by outscale-rce — APPROVED** [review]

**Pull request merged**